src/core/selfcare.tf

```diff
@@ -208,14 +213,15 @@ module "appservice_selfcare_be" {
     SERVICE_PRINCIPAL_TENANT_ID = data.azurerm_client_config.current.tenant_id
     USE_SERVICE_PRINCIPAL       = "1"
 
-    FRONTEND_URL                          = "https://${local.selfcare.frontend_hostname}"
-    BACKEND_URL                           = "${local.selfcare.backend_hostname}"
-    LOGIN_URL                             = "https://${local.selfcare.frontend_hostname}/login"
-    FAILURE_URL                           = "https://${local.selfcare.frontend_hostname}/500.html"
-    SELFCARE_LOGIN_URL                    = "https://uat.${var.selfcare_external_hostname}/auth/login"
-    SELFCARE_IDP_ISSUER                   = "api.${var.selfcare_external_hostname}"
-    SELFCARE_IDP_ISSUER_JWT_SIGNATURE_KEY = data.azurerm_key_vault_secret.selfcare_selfcare_idp_issuer_jwt_signature_key.value # todo data.http.selfcare_well_known_jwks_json.body
-    JWT_SIGNATURE_KEY                     = trimspace(module.selfcare_jwt.jwt_private_key_pem) # to avoid unwanted changes
+    FRONTEND_URL                          = "https://${local.selfcare_io.frontend_hostname}"
+    BACKEND_URL                           = "${local.selfcare_io.backend_hostname}"
+    LOGIN_URL                             = "https://${local.selfcare_io.frontend_hostname}/login"
+    FAILURE_URL                           = "https://${local.selfcare_io.frontend_hostname}/500.html"
+    SELFCARE_LOGIN_URL                    = "${local.selfcare.base_url}/auth/login"
```

Comment:
not blocking
only for coherence change in `local.selfcare.hostname` and set `SELFCARE_LOGIN_URL=https://...`